### ISSUE TITLE
[PHP] Error: Undefined index: comments at /var/www/html/apps/music/lib/Utility/DetailsHelper.php#40

### ISSUE THREAD

**Issue by mikaelkundert** [owncloud/music#989]
Picked up this error, but no idea how.

```
[PHP] Error: Undefined index: comments at /var/www/html/apps/music/lib/Utility/DetailsHelper.php#40

GET /apps/music/api/file/29908/details
from xxx.xxx.xxx.xxx by <user> at 2022-06-16T11:27:02+00:00
```

```
{"reqId":"WJM23pnPZ1z9hCn6DFuM","level":3,"time":"2022-06-16T11:27:02+00:00","remoteAddr":"xxx.xxx.xxx.xxx","user":"<user>","app":"PHP","method":"GET","url":"/apps/music/api/file/29908/details","message":"Undefined index: comments at /var/www/html/apps/music/lib/Utility/DetailsHelper.php#40","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:101.0) Gecko/20100101 Firefox/101.0","version":"23.0.5.1","exception":{"Exception":"Error","Message":"Undefined index: comments at /var/www/html/apps/music/lib/Utility/DetailsHelper.php#40","Code":0,"Trace":[{"file":"/var/www/html/apps/music/lib/Utility/DetailsHelper.php","line":40,"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"file":"/var/www/html/apps/music/lib/Controller/ApiController.php","line":288,"function":"getDetails","class":"OCA\\Music\\Utility\\DetailsHelper","type":"->"},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":217,"function":"fileDetails","class":"OCA\\Music\\Controller\\ApiController","type":"->"},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":126,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":157,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/html/lib/private/Route/Router.php","line":302,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/html/lib/base.php","line":1008,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/html/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/html/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"},"id":"62ab311594f37"}
```

Nextcloud Hub II, 23.0.5
Music 1.5.2

**Comment by paulijar** [comment]
I suspect that you have tried to open the details pane within the Music app for an audio file which is somehow corrupted. Or it might be enough if the file is otherwise fine but just doesn't contain any kind of metadata. The Music app actually tries to handle this gracefully, but there is a small flaw on how this is done. To be fixed.

**Comment by paulijar** [comment]
This is now fixed in Music v1.6.0.
